### ISSUE TITLE
Have you finished the book?

### ISSUE THREAD

**Issue by mcbucks** [cfsamson/book-exploring-epoll-kqueue-iocp#4]

**Comment by cfsamson** [comment]
Yeah, but it’s a gitbook so the rendered version is found here: https://cfsamsonbooks.gitbook.io/epoll-kqueue-iocp-explained/

I should probably link to it in the readme as well so it’s easier to find.
